cms/djangoapps/contentstore/views/tests/test_course_marketing_site.py

```diff
@@ -0,0 +1,74 @@
+"""
+Unit tests for course settings UI in Studio with marketing site flags.
+"""
+
+import mock
+
+from contentstore.tests.utils import CourseTestCase
```

Comment:
@bderusha there are already test cases related to the marketing site in test_course_settings.py. Can you move your test cases there?